docs/source/api/apollo-server.mdx

```diff
@@ -228,6 +229,26 @@ Available in Apollo Server v3.4.0 and later.
 </td>
 </tr>
 
+<tr>
+<td>
+
+##### `cache`
+
+`KeyValueCache | "bounded"`
```

Comment:
`KeyValueCache` or `String` ("bounded")I believe this would need to be

```
`KeyValueCache` or `String` ("bounded")
```
to match the established table styling.

Reply:
I think the original is better. It really is only the string "bounded".